src/components/SubscriberListSelector/NoList/NoList.js

```diff
@@ -0,0 +1,45 @@
+import { useIntl } from 'react-intl';
+import { FormattedMessageMarkdown } from '../../../i18n/FormattedMessageMarkdown';
+import React from 'react';
+
+export const NoList = ({ onCancel }) => {
```

Comment:
I think this component can be general. Might need to be used in another context. I think to this component should have props: `onCancel, onAction, logo, title, description, paragraph, cancelButtonText, actionButtonText` and should be in `shared` folder.

Reply:
Sounds good to me, we would have to change the name too, a more generic name, what names can you think of?

Reply:
What do you think of these names: `Confirmation`, `ConfirmationBox` or `ConfirmationDialog`? @jcamposmk @cbernat

Reply:
I like something like <NoResultsFound /> . ConfirmationBox it sounds very tied to the implementation to me.